app/components/price-graph/price-graph.tsx

```diff
@@ -49,6 +49,7 @@ export const PriceGraphDataInjected = () => {
 
   const { error, loading, data, refetch } = useBtcPriceListQuery({
     fetchPolicy: "no-cache",
+    variables: { range: graphRange },
```

Comment:
the value is passed there already: https://github.com/GaloyMoney/galoy-mobile/pull/979/files#diff-d75c8934e23c87b556deda51f102e3d3045a1da88266c8a2391ba51c8a130873R68

if there is an issue with this function, line 68, 72, ... probably needs some refactoring. ie: update the state of `graphRange` with `setGraphRange` instead of calling refetch directly?